AirCasting/SearchAndFollow/CompleteScreen/Views/CompleteScreenViewModel.swift

```diff
@@ -186,7 +186,15 @@ class CompleteScreenViewModel: ObservableObject {
                                                                                      high: streamLocalData?.thresholdHigh ?? 0,
                                                                                      veryHigh: streamLocalData?.thresholdVeryHigh ?? 0))})
                     self.session.stream = []
-                    sessionStream.forEach { self.session.stream.append($0) }
+                    var sortedStreams = [PartialExternalSession.Stream]()
+                    #warning("-- think what to do here when fixing thresholds 🚨")
+                    streamSortedHardcodedData.forEach { sorted in
+                        if let first = sessionStream.first(where: { Self.getSensorName($0.sensorName) == Self.getSensorName(sorted.sensorName ?? "") }) {
+                            sortedStreams.append(first)
```

Comment:
What do you think about removing `sortedStreams ` and instead of appending there directly to `self.session.stream.append(first)`? 🤔